components/content/zh/page-sections.md

```diff
@@ -0,0 +1,3 @@
+---
+edit: '编辑'
```

Comment:
@kentcdodds  yes, and things need to be changed in all language versions translation....how about open an issue to let them known?